### PR TITLE
Testimonial card

### PR DESCRIPTION
**Pull request by samdkuo**

### CODE REVIEW

**Comment by douglaschan32167** [comment]
Thanks for working on this!
I like the smaller, square card but I'm not sure how I feel about ellipsis truncation with no option to click to see the full text. I wonder what other people think.

**Comment by richyen** [comment]
Re-assigning for visibility

**Comment by douglaschan32167** [comment]
Thanks for working on this Sam! I like the way this works. Just a few questions:
1). Can you merge master again? Looks like you last merged during the time where docker-compose up was broken. Not your fault, just want to make sure that doesn't make it's way back into our code base.
2). I notice that the size the cards start out at the the size they contract to are different. Would it be possible to start them out at the contracted size? 
image

File: frontend/src/components/TestimonialCards/TestimonialCards.jsx [hunk 1]
```diff
@@ -66,53 +78,59 @@ const Container = ({ children }) => {
...
 `;
 
 const MobileRatio = styled(DesktopRatio)`
-  padding-top: 285px;
   width: 100%;
+  max-width: 100%;
   margin: 1em;
 `;
 
 const CardRatio = ({ children }) => {
```
Comment (dbtung) [inline frontend/src/components/TestimonialCards/TestimonialCards.jsx:93]: I was a bit confused by the naming - what exactly do you mean by `Ratio`? Seems like more appropriate name might be `CardContainer`?

File: frontend/src/components/TestimonialCards/TestimonialCards.jsx [hunk 2]
```diff
@@ -66,53 +78,59 @@ const Container = ({ children }) => {
...
   background-color: #fafafa;
   padding: 0.25em 1em 1.5em 1em;
   margin: 0;
   font-family: ${theme.fonts.PTSerif};
-  &:hover {
+  /* &:hover {
     background-color: white;
-  }
+  } */
 `;
```
Comment (dbtung) [inline frontend/src/components/TestimonialCards/TestimonialCards.jsx:126]: Similar to above, can we get rid of commented styles if they aren't needed?

File: frontend/src/components/TestimonialCards/TestimonialCards.jsx [hunk 3]
```diff
@@ -180,25 +196,51 @@ const ProfileImage = styled.img`
...
+    font-size: 12;
+    padding: 0;
+    margin: 0;
+    background: none;
+    color: gray;
+  }
+`;
+
+const TestimonialQuote = ({ children }) => {
+  const [size, setSize] = useState(3);
```
Comment (dbtung) [inline frontend/src/components/TestimonialCards/TestimonialCards.jsx:210]: Maybe rename size to numLines?

**Comment by douglaschan32167** [comment]
I can't open and close each card more than once before refreshing the page, but I don't mind pushing this through and filing a bug since this has been open for a while. I'll see what other people think.

**Comment by samdkuo** [comment]
ok took a look at it the code, it should work now

**Comment by douglaschan32167** [comment]
One last change: can you remove the images (so we don't have to check them in) and instead use 
https://ismp-us-east-1.s3.amazonaws.com/testimonials/headshots/gabi.jpg
https://ismp-us-east-1.s3.amazonaws.com/testimonials/headshots/stella.jpg

**Pull request merged**